java-connectors/kafka-connect-azure-eventhubs/src/main/java/io/lenses/java/streamreactor/connect/azure/eventhubs/config/AzureEventHubsConfig.java

```diff
@@ -1,154 +1,185 @@
-          AzureEventHubsConfigConstants.CLIENT_ID_DOC,
-          CONNECTION_GROUP,
-          9,
-          ConfigDef.Width.MEDIUM,
-          AzureEventHubsConfigConstants.CLIENT_ID
-      ).define(AzureEventHubsConfigConstants.CLIENT_SECRET,
-          Type.STRING,
-          AzureEventHubsConfigConstants.OPTIONAL_EMPTY_DEFAULT,
-          Importance.HIGH,
-          AzureEventHubsConfigConstants.CLIENT_SECRET_DOC,
-          CONNECTION_GROUP,
-          10,
-          ConfigDef.Width.MEDIUM,
-          AzureEventHubsConfigConstants.CLIENT_SECRET
-      ).define(AzureEventHubsConfigConstants.TENANT_ID,
-          Type.STRING,
-          AzureEventHubsConfigConstants.OPTIONAL_EMPTY_DEFAULT,
-          Importance.LOW,
-          AzureEventHubsConfigConstants.TENANT_ID_DOC,
-          CONNECTION_GROUP,
-          11,
-          ConfigDef.Width.MEDIUM,
-          AzureEventHubsConfigConstants.TENANT_ID
-      ).define(AzureEventHubsConfigConstants.USER_INFO,
-          Type.STRING,
-          AzureEventHubsConfigConstants.OPTIONAL_EMPTY_DEFAULT,
-          Importance.LOW,
-          AzureEventHubsConfigConstants.USER_INFO_DOC,
-          CONNECTION_GROUP,
-          12,
-          ConfigDef.Width.MEDIUM,
-          AzureEventHubsConfigConstants.USER_INFO
-      ).define(AzureEventHubsConfigConstants.KCQL_CONFIG,
-          Type.STRING,
-          AzureEventHubsConfigConstants.KCQL_DEFAULT,
-          Importance.HIGH,
-          AzureEventHubsConfigConstants.KCQL_DOC,
-          "Mappings",
-          1,
-          ConfigDef.Width.LONG,
-          AzureEventHubsConfigConstants.KCQL_CONFIG
-      ).define(AzureEventHubsConfigConstants.INCLUDE_HEADERS,
-          Type.BOOLEAN,
-          AzureEventHubsConfigConstants.INCLUDE_HEADERS_DEFAULT,
-          Importance.MEDIUM,
-          AzureEventHubsConfigConstants.INCLUDE_HEADERS_DOC,
-          "Mappings",
-          2,
-          ConfigDef.Width.LONG,
-          AzureEventHubsConfigConstants.INCLUDE_HEADERS
-      ).define(
-          AzureEventHubsConfigConstants.NBR_OF_RETRIES_CONFIG,
-          Type.INT,
-          AzureEventHubsConfigConstants.NBR_OF_RETRIES_CONFIG_DEFAULT,
-          Importance.LOW,
-          AzureEventHubsConfigConstants.NBR_OF_RETRIES_CONFIG_DOC,
-          "Error",
-          1,
-          ConfigDef.Width.LONG,
-          AzureEventHubsConfigConstants.NBR_OF_RETRIES_CONFIG
-      );
+  static ConfigDef configDefinition;
+
+  static {
+    ConfigDef kafkaConsumerConfigToExpose = getKafkaConsumerConfigToExpose();
+    configDefinition = new ConfigDef(kafkaConsumerConfigToExpose)
+        .define(AzureEventHubsConfigConstants.EVENTHUB_NAME,
+            Type.STRING,
+            Importance.HIGH,
+            AzureEventHubsConfigConstants.EVENTHUB_NAME_DOC,
+            CONNECTION_GROUP,
+            1,
+            ConfigDef.Width.LONG,
+            AzureEventHubsConfigConstants.EVENTHUB_NAME
+            //CONFIGS BELOW IDENTIFIED AS NOT NEEDED (for now)
```

Comment:
let's remove the commented-out code.

Reply:
Happy to do so! Didn't want to remove them straight away as they were present in original story (LC-107)